pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -168,4 +168,8 @@ func addAlertMigrations(mg *Migrator) {
 	mg.AddMigration("Remove unique index org_id_name", NewDropIndexMigration(alert_notification, &Index{
 		Cols: []string{"org_id", "name"}, Type: UniqueIndex,
 	}))
+
+	mg.AddMigration("Add secure settings data column", NewAddColumnMigration(alert_notification, &Column{
```

Comment:
Good to include table name to make it unique (just in case).
```suggestion
	mg.AddMigration("Add column secure_settings in alert_notification", NewAddColumnMigration(alert_notification, &Column{
```